### PR TITLE
chore: workflow for automatically updating files during a release

### PR DESCRIPTION
**Pull request by dsherret**
This adds a new workflow that we can dispatch which will run `deno task build` then open a PR.

Two commits:

1. First switches to using my new library.
2. Second adds the workflow.

### CODE REVIEW

File: build.ts [hunk 1]
```diff
@@ -28,32 +29,19 @@ interface GitHubRelease {
...
-const gitHubAPIFetchOptions = {
-  headers: {
-    accept: "application/vnd.github.v3+json",
-  },
+const gitHubAPIHeaders = {
+  accept: "application/vnd.github.v3+json",
 } as const;
 
-console.log(`${colors.bold(colors.green("Building"))} deploy_doc...`);
+$.logTitle("Building deploy_doc...");
```
Comment (dsherret) [inline build.ts:36]: Still looks like before:

image

File: build.ts [hunk 2]
```diff
@@ -28,32 +29,19 @@ interface GitHubRelease {
...
-const gitHubAPIFetchOptions = {
-  headers: {
-    accept: "application/vnd.github.v3+json",
-  },
+const gitHubAPIHeaders = {
+  accept: "application/vnd.github.v3+json",
 } as const;
 
-console.log(`${colors.bold(colors.green("Building"))} deploy_doc...`);
+$.logStep("Building deploy_doc...");
```
Comment (dsherret) [inline build.ts:36]: Still works (tested this new build script on an old commit of this repo):

image

**Review by bartlomieju — APPROVED** [review]
LGTM, remember to update `cut_a_release.md` doc in `deno` repo

**Comment by dsherret** [comment]
Yup, will do.

**Pull request merged**